neuralnetwork/genome.js

```diff
@@ -184,6 +184,53 @@ class Genome {
         return new Genome({ nodeGenes: copiedNodes, connectionGenes: copiedConnections });
     };
 
+    static numExcess = (genomeA, genomeB) => Math.abs(genomeA.innovationSet().maxInnovation - genomeB.innovationSet().maxInnovation);
```

Comment:
are innovation numbers always continuously increasing with no gaps?
what if the two genomes are 1-2-3 and 1-2-3-6? there is only one excess gene but wouldnt this say it has three?
your algorithm seems to assume all genome innovation numbers go somehting like 1-2-3-4-5-6-7...